frontend/src/metabase/embedding/embedding-hub/components/StepperWithCards/StepperWithCards.tsx

```diff
@@ -89,109 +93,146 @@ export const StepperWithCards = ({ steps }: { steps: StepperStep[] }) => {
             className={cx(S.step, isDone && S.stepDone)}
             icon={isDone ? <Icon name="check" /> : undefined}
             description={
-              <Grid>
-                {step.cards.map((card) => {
-                  const onClick =
-                    card.clickAction?.type === "click"
-                      ? card.clickAction.onClick
-                      : undefined;
-
-                  const isNextCard = nextCard && nextCard.id === card.id;
-
-                  return (
-                    <Grid.Col
-                      span={{ xs: 12, md: card.optional ? 4 : 8 }}
-                      key={card.id}
-                    >
-                      <CardAction card={card}>
-                        <Card
-                          h="100%"
-                          mih="8.75rem"
-                          className={cx(S.stepCard, {
-                            [S.lockedStepCard]: card.locked,
-                            [S.nextStepCard]: isNextCard,
-                          })}
-                          component={onClick ? "button" : undefined}
-                          onClick={onClick}
-                          disabled={card.locked}
-                          data-testid={`step-card-${card.id}`}
-                          data-next-step={isNextCard}
-                        >
-                          <Stack justify="space-between" h="100%">
-                            <Stack gap="xs" h="100%">
-                              <Text
-                                size={card.optional ? "md" : "lg"}
-                                fw="bold"
-                                c={
-                                  card.done
-                                    ? "var(--mb-color-text-secondary)"
-                                    : "var(--mb-color-text-primary)"
-                                }
-                              >
-                                {card.title}
-                              </Text>
-
-                              <Text
-                                c="var(--mb-color-text-secondary)"
-                                size="sm"
-                                lh="lg"
-                              >
-                                {card.description}
-                              </Text>
-                            </Stack>
+              <div>
```

Comment:
If you hide whitespace changes, nothing here changes except that it's wrapped with a `<div>` so I can add the bottom alert.